cmake-tool/helpers/application_settings.cmake

```diff
@@ -56,6 +59,9 @@ function(ApplyData61ElfLoaderSettings kernel_platform kernel_sel4_arch)
         endif()
         set(IMAGE_START_ADDR 0x10000000 CACHE INTERNAL "" FORCE)
     endif()
+    if(KernelPlatformStar64)
+        set(IMAGE_START_ADDR 0x60000000 CACHE INTERNAL "" FORCE)
```

Comment:
I'm not sure what I would say because it comes from the same place as all the other addresses in that file - somewhere in RAM that doesn't overlap with anything.

Reply:
Ah ok, could say this for the whole section then, that it's an arbitrary choice unless stated otherwise. That answer feels lika a deja-vu somehow.